src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -91,12 +91,10 @@ function VAFacilityPageV2({
     const facility = facilities.find(f => f.id === newData.vaFacility);
     const vaParent = getParentOfLocation(parentFacilities, facility)?.id;
 
-    if (!!facility && !!vaParent) {
```

Comment:
This conditional was preventing errors from surfacing.  Although Sacramento should be a unique scenario, removing this conditional so we can track any errors related to the radio onChange.